.changeset/cuddly-mugs-repeat.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I don't think we need empty changesets... we can just merge with a failing changeset check

Reply:
It's nice to see the overall green checkmark though, right? And they'll get deleted automatically on next publish?

(Not that we ever get a green checkmark, because our IE tests are permanently failing?)